microsite/src/pages/home/hubSpotNewAdoptersForm.module.scss

```diff
@@ -4,6 +4,8 @@
   display: flex;
   position: fixed;
   transform: translateY(-50%);
+  margin-top: 44px;
```

Comment:
What's this magic number? Can we have a comment? :pray:

Reply:
That's what the global header height happens to be. Sure I can add a comment for that

Reply:
made it a variable instead!